src/views/open_data.dot

```diff
@@ -0,0 +1,8 @@
+<div class="open_data">
+  {{= _('Qwant use OpenStreetMap') }}
+  <i class="icon-chevron-right open_data__chevron_fix"></i>
+  <i class="icon-chevron-right"></i>
+  <a class="open_data__link" href="https://learnosm.org/" target="_blank">
+    {{= _('KNOW MORE', 'open data') }}
```

Comment:
I am not sure about the wording of this panel.
And maybe we would like to add a button to close this banner ?

Reply:
ping @nlehuby

Reply:
Maybe "Qwant Maps uses OpenStreetMap. Learn More" ?